packages/dashboard/src/dashboard-actions/dashboardReducer.spec.ts

```diff
@@ -1,17 +1,283 @@
 import { dashboardReducer } from './dashboardReducer';
-import { onMoveAction } from './actions';
-import { MOCK_EMPTY_DASHBOARD } from '../testing/mocks';
+import { onCreateAction, onMoveAction, onPasteAction } from './actions';
+import { DashboardReducerState, DashboardConfiguration } from '../types';
+import { onDeleteAction, onCopyAction, onEditCellSizeAction, onEditWidthAction } from '../dashboard-actions/actions';
+import {
+  MOCK_EMPTY_DASHBOARD,
+  MOCK_KPI_WIDGET,
+  MOCK_LINE_CHART_WIDGET,
+  MockDashboardFactory,
+  MockWidgetFactory,
+  dashboardConfig,
+} from '../testing/mocks';
+import { move } from './move';
+import { deleteWidgets } from './delete';
+import { trimWidgetPosition } from '../components/iot-dashboard/trimWidgetPosition';
 
-it('returns empty dashboard configuration when provided an empty dashboard', () => {
-  expect(
-    dashboardReducer(
-      MOCK_EMPTY_DASHBOARD,
+describe('MOVE', () => {
+  it('returns empty dashboard configuration when provided an empty dashboard', () => {
+    expect(
+      dashboardReducer(
+        {
+          dashboardConfiguration: MOCK_EMPTY_DASHBOARD,
+          intermediateDashboardConfiguration: undefined,
+          selectedWidgetIds: [],
+          numTimesCopyGroupHasBeenPasted: 0,
+          copyGroup: [],
+          stretchToFit: true,
+          width: 1000,
+          cellSize: 30,
+        },
+        onMoveAction({
+          widgetIds: [],
+          cellSize: 10,
+          position: { x: 10, y: 10 },
+          prevPosition: { x: 20, y: 10 },
+          isActionComplete: true,
+        })
+      )
+    ).toEqual({
+      dashboardConfiguration: MOCK_EMPTY_DASHBOARD,
+      intermediateDashboardConfiguration: undefined,
+      selectedWidgetIds: [],
+      numTimesCopyGroupHasBeenPasted: 0,
+      copyGroup: [],
+      stretchToFit: true,
+      width: 1000,
+      cellSize: 30,
+    });
+  });
```

Comment:
minor general comment: new lines can help break up 'blocks of code' for easier readability